### ISSUE TITLE
Unable to debug remotely

### ISSUE THREAD

**Issue by a11enhuang** [alibaba/dragonwell11#347]
My application is running in kubernetes, when I try to use remote debug, I found that da cannot be connected。
Here are the startup parameters I added:
```
-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:11899
```
This is the error response I get:
```
Unable to open debugger port (xxx:11899): java.net.SocketException "Connection reset"
```

**Comment by D-D-H** [comment]
Is it a Dragonwell-specific problem? Have you tried using other JDKs to see if it works well?

**Comment by a11enhuang** [comment]
Yes, I used bitnami/java:11.0.17-7 as the base scene and it connected successfully.

**Comment by D-D-H** [comment]
Sorry, I cannot reproduce this problem in my env.
Would it be possible for you to provide more details about your env? e.g. Dockerfile
